src/build.sh

```diff
@@ -154,6 +164,17 @@ function kernel_build()
   return "$ret"
 }
 
+# This function adds the build clean option
+# integrated to env in order to clean the kernel.
```

Comment:
How about something like this?

This function runs the 'make clean' command under the hood, with the advantage of checking if the user is using an env or not. In other words, it integrates env with the clean option.

Also, document the output_kbuild_flag parameter and flag.

Reply:
I think it's a good idea.

As for the documentation part of the `output_kbuild_flag` parameter, I think it will not be necessary, as I will remove this variable from the function, since it is global. only the flag variable will remain for the `cmd_manager` function